### DISCUSSION TITLE
Dealing with a large data set and ASReview

### DISCUSSION THREAD

**Discussion by Taz-iz** [asreview/asreview#1805] [Q&A]
I am working on a literature review and have a very large data set (approximately 11 940 articles) that I am currently screening using ASReview. I find that I have a lot of relevant articles than irrelevant and so I would like to know if I can tighten my selection criteria I will have to label some of the articles that are marked as relevant to irrelevant, would that have any changes to the algorithm with the same project. For example, if I have found 900 articles relevant and mark 700 as irrelevant how would that affect the algorithim

**Comment by jteijema** [comment]
Well, keep in mind that the rate at which you find relevant records will decline as you keep going. I'd consider to keep going until this starts happening.

However, if you do want to change the selection criteria, and change some documents' label, then that will give no problems for the classifier. The classifier is trained only on the latest set of documents. If you change some documents, you will get a brand new classifier!
